app/views/pages/ivFailure/lockedOut.scala.html

```diff
@@ -17,21 +17,22 @@
 @import common.Links
 @import controllers.PlaLanguageController
 @import play.api.Application
-@import config.FrontendAppConfig
 
 @this(
- main_template: main_template)
+ layout: Layout)
 
-@()(implicit request: Request[_], messages: Messages, application: Application, context: config.PlaContext, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, appConfig: FrontendAppConfig)
+@()(implicit request: Request[_], messages: Messages, application: Application)
 
 @lc = @{Application.instanceCache[PlaLanguageController].apply(application) }
 
-@main_template(title = Messages("pla.lockedOut.title"), bodyClasses = None, showLogout = false) {
+@layout(
+ pageTitle = Messages("pla.lockedOut.title")
+) {
 
-<h1 class="heading-large">@Messages("pla.lockedOut.pageHeading")</h1>
+<h1 class="govuk-heading-xl">@Messages("pla.lockedOut.pageHeading")</h1>
 
-<p>@Messages("pla.lockedOut.paraOne")</p>
+<p class="govuk-body">@Messages("pla.lockedOut.paraOne")</p>
 
-@Html(Messages("pla.lockedOut.paraTwo", Links.contactInfo))
+ @Html(Messages("pla.lockedOut.paraTwo", Links.contactInfo))
```

Comment:
The html for this `<a>` element could be moved into this template (rather than the messages). It could also have `rel="noopener noreferrer"` added to its attributes as it opens in a new tab.